src/crewai/crew.py

```diff
@@ -623,6 +648,36 @@ def _execute_tasks(
                 )
             self._log_task_start(task, agent_to_use)
 
+            if isinstance(task, ConditionalTask):
+                if futures:
+                    task_outputs.extend(
```

Comment:
Why are you extending task_outputs? I believe it should be like this right?

`task_outputs = self._process_async_tasks(futures, was_replayed)`